### PR TITLE
Add pipeline disabler function to open sourced package

### PR DESCRIPTION
**Pull request by real-laser**
Contents of the README file is the only real change

### CODE REVIEW

File: aws-codepipeline-transition-disabler/pom.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.replyyes</groupId>
+  <artifactId>aws-codepipeline-transition-disabler</artifactId>
+  <version>0.1.0</version>
+  <packaging>jar</packaging>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>AWS Lambda function for automatically disabling CodePipelines after deployments. </description>
```
Comment (trevorinman) [inline aws-codepipeline-transition-disabler/pom.xml]: Remove the extra space at the end of the sentence.

File: aws-codepipeline-transition-disabler/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+1. Make sure that your beanstalk environment has notifications enabled by clicking on the "Configuration" page and selecting the "Notifications" tab. Enter an email address and click apply.
+
+## Lambda Function
+
+Create a new Lambda function to run the disabler code:
+
+1. Visit https://console.aws.amazon.com/lambda/home
+2. If this will be your first Lambda function, click "Get Started", otherwise click "Create a Lambda Function".
+3. Select "Blank Function".
+4. On the "Select Triggers" page, click next (we will setup the triggers later).
```
Comment (trevorinman) [inline aws-codepipeline-transition-disabler/README.md]: "setup" -> "set up"

File: aws-codepipeline-transition-disabler/.gitignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/target/
```
Comment (coffeechris) [inline aws-codepipeline-transition-disabler/.gitignore]: thoughts about having .gitkeep files in the `/src/main/resources`, `/src/test/java`, and `/src/test/resources`

Comment (real-laser) [reply aws-codepipeline-transition-disabler/.gitignore]: I've added one for `/src/test/java` but my local version doesn't have either resources directory and it builds fine, so I'm thinking those are fine without.

File: aws-codepipeline-transition-disabler/pom.xml [hunk 4]
```diff
@@ -0,0 +1,94 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.replyyes</groupId>
+  <artifactId>aws-codepipeline-transition-disabler</artifactId>
+  <version>0.1.0</version>
```
Comment (coffeechris) [inline aws-codepipeline-transition-disabler/pom.xml]: I would make this 1.0.0

File: aws-codepipeline-transition-disabler/.gitignore (hunk 3 above)
Comment (trevorinman) [reply aws-codepipeline-transition-disabler/.gitignore]: What's the point of adding .gitkeep files for these folders? They're not required.

Comment (coffeechris) [reply aws-codepipeline-transition-disabler/.gitignore]: Makes it easier to add UTs and resources later.

**Review by coffeechris — APPROVED** [review]

**Pull request merged**